script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -137,90 +137,90 @@ Hey, why does the hospital have cracks everywhere?"
+Sort of like the mark on my chest that was still healing after surgery. And just like that, could be torn apart with a touch.
+Because otherwise, there was no way a kid could slice through a wall with strength alone.
 <Page46>
 "--- #---!"
-The nurse is yelling at me again.
+The nurse was yelling at me again.
 After staring at the unevenly cut hole in the wall, she glared at me.
 <Page47>
-The shrieking gets louder and louder.
-My hearing has gotten hazy, so I don't find it too obnoxious.
-The never ending noise reminds me of a cicada lit on fire; it's making me uncomfortable.
+The shrieking got louder and louder.
+My hearing had gotten hazy, so I didn't find it too obnoxious.
+The never ending noise reminded me of a cicada lit on fire; it was making me uncomfortable.
 <Page48>
 Everything is falling apart.
-Just opening my eyes, just being alive, is sickening enough to make me want to disappear.
+Just opening my eyes, just being alive, was sickening enough to make me want to disappear.
 <Page49>
 ......Ah. I didn't realize it until now.
-This world is full of stitches.
+This world was full of stitches.
 I had no idea it could be torn apart that easily.
 <Page50>
-No one can see it.
-I guess that's how they can remain so calm.
-They don't even notice how messed up this world is, at all.
-They don't know their everyday life is full of holes.
-But I can see it.
-And it scares me so much, I can't even walk.
+No one could see it.
+I guessed that was how they could remain so calm.
+They didn't even notice how messed up this world was, at all.
+They didn't know their everyday life was full of holes.
+But I could see it.
+And it scared me so much, I couldn't even walk.
 <Page51>
-Speaking of which, my ears have been frozen since then, and it's gotten quieter despite the noise.
-Only the wound in my chest reverberates like a siren.
+Speaking of which, my ears had been frozen since then, and it had gotten quieter despite the noise.
+Only the wound in my chest reverberated like a siren.
 But "when" was "that"?
-I try to recall, but it's no use. I scratch my head, but even that doesn't help. I'm sure I must've forgotten about it because it wasn't very important. Like a faucet that kept spilling water no matter how hard you close it.
+I tried to recall, but it was no use. I scratched my head, but even that didn't help. I was sure I had forgotten about it because it wasn't very important. Like a faucet that kept spilling water no matter how hard you close it.
 <Page52>
 ......Ah, I see.
-That explains everything.
-It makes sense there isn't a mirror here.
+That explained everything.
+It made sense there wasn't a mirror here.
 If that were the case, I would have ■ before anything else.
-I'm sure I'd crack up laughing.
-In other words, it's not the world around me that's breaking down.
-I'm the only one that's broken.
+I was sure I would have been cracking up laughing.
+In other words, it wasn't not the world around me that was breaking down.
```

Comment:
`wasn't not` -> either just `wasn't` or `was not`

Reply:
I'm pretty sure this is a typo lol